tests/test_wave_05.py

```diff
@@ -74,40 +69,45 @@ def test_create_goal(client):
     assert "goal" in response_body
     assert response_body == {
         "goal": {
-            "id": 1,
+            "goal_id": 1,
             "title": "My New Goal"
         }
     }
 
-
-@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
+    
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goals/1", json={"title": "Be healthier"})
+    response_body = response.get_json()
 
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
-    # ---- Complete Assertions Here ----
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+        "goal":{
+            "goal_id":1,
+            "title":"Be healthier"
+        }
+    }
+    #check if it was updated
+    goal = Goal.query.get(1)
+    assert goal.title == "Be healthier"
```

Comment:
Good test design, especially with going to the DB to verify that it was updated there too.